core/src/main/scala/com/softwaremill/events/model.scala

```diff
@@ -100,7 +100,20 @@ case class PartialEventWithId[U, T](id: Long, eventType: String, aggregateType:
 }
 
 case class StoredEvent(id: Long, eventType: String, aggregateType: String, aggregateId: Long, aggregateIsNew: Boolean,
-  created: OffsetDateTime, userId: Long, txId: Long, eventJson: String)
+  created: OffsetDateTime, userId: Long, txId: Long, eventJson: String){
+
+  def toEvent(clsOpt: Option[Class[_]]): Event[Any] = {
+    //TODO: capture formats with event, maybe in registry
+
+    clsOpt match {
+      case Some(clazz) =>
```

Comment:
obviously it shouldn't happen but for that case I need to return any Event[Any], would it be fine to return an empty Event?